### PR TITLE
7 post article comment

### PR DESCRIPTION
**Pull request by faithcollison**

### CODE REVIEW

File: __tests__/index.test.js [hunk 1]
```diff
@@ -107,3 +107,49 @@ describe('GET /api/articles', () => {
...
+            body: "10/10 enjoyed, will be reading this again."
+        };
+        return request(app)
+        .post("/api/articles/1/comments")
+        .send(newComment)
+        .expect(400)
+        .then((response) => {
+            expect(response.body.msg).toBe('Bad request');
+        });
+    });
```
Comment (MRombi) [inline __tests__/index.test.js:201]: We should add a test for when we are passing an username that is not found(404), a newComment that is missing either the body or the username (400), and maybe one with additional unrelated properties (200) like {
            body: "10/10 enjoyed, will be reading this again.",
            username: "butter_bridge",
            fruit: "banana"
        };

**Review by MRombi — CHANGES_REQUESTED** [review]
Well done on this first attempt to this endpoint, please action on the feedback and resubmit 🙂

File: __tests__/index.test.js (hunk 1 above)
Comment (faithcollison) [reply __tests__/index.test.js:201]: the error for passing an invalid username(username not found) is coming up as a 400 error (has error code), is this ok? is there a way to manually change a psql error to a custom error?

Comment (faithcollison) [reply __tests__/index.test.js:201]: "maybe one with additional unrelated properties (200) like {
body: "10/10 enjoyed, will be reading this again.",
username: "butter_bridge",
fruit: "banana" <<<< i am unsure if this is an error test or a happy test? when i tried it , it didn't throw an error, not sure how to make it do this?

**Comment by saimakhanom** [comment]
All good to merge!

**Pull request merged**